Magefile.go

```diff
@@ -32,7 +32,7 @@ func Drone() error {
 		return err
 	}
 
-	if err := sh.RunV("drone", "--server", "https://drone.grafana.net", "sign", "--save", "grafana/grafana-plugin-sdk-go"); err != nil {
+	if err := sh.RunV("drone", "--server", "https://drone.grafana.net", "sign", "--save", "grafana/grafana-aws-sdk-react"); err != nil {
```

Comment:
Do we need to have this file for Drone?

Reply:
Not really, it was just out of convenience to not having to remember the whole drone command but I can just add it to the docs.